### ISSUE TITLE
Some color issues

### ISSUE THREAD

**Issue by jerryn70** [SimpleMobileTools/Simple-Gallery#251]
When I choose background color and primary color as black the filter selection cannot be viewed correctly
![screenshot_20170516-224113](https://cloud.githubusercontent.com/assets/23030460/26119078/ad3be66a-3a89-11e7-8273-a14135cf6134.png)

**Comment by tibbi** [comment]
Marking it as a bug, but it wont be changed anytime soon. Get some nice color for contrast :)

**Comment by jerryn70** [comment]
Ya I know it's not a bug . But I just reported it because when I made that settings i got that problem. 

Hey what about making an option to adopt primary color to navigation bar as an option ?
So if we set primary color to red then the navigation bar also get red color . Just an option like "adopt primary color to navigation bar" is enough. So that the color changes only if we enabled that option, making default is not a good idea.
